src/analysis/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#automatically generated CMakeLists.txt, may not work!
```

Comment:
@Iximiel are these file lists automatically generated? If so, is it possible not to include them in the repository?

Ideally, the build system will just use all the files in that directory. Can cmake do this?

Reply:
the base file is "automatically generated" by a homemade sh script (that I'm in doubt of including or not in the repository), but the ADDMODULEDEPENDENCIES macro arguments are them manually changed.
This because I wanted to reduce the "dependencies entanglement" and some circular dependency tree that exist if we use the list of dependencies in the "USE=" from the original Makefile (like core and tools).
I forgot to remove the lines "#automatically generated CMakeLists.txt, may not work!".